scripts/build-win.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs')
+
+const dir = './apks'
+
+if (!fs.existsSync(dir)){
```

Comment:
Prefer async calls to sync ones. Check https://github.com/appium/appium-uiautomator2-driver/blob/master/scripts/reset.js for example

Reply:
Done